frontend/packages/console-shared/src/components/shortcuts/ShortcutTable.scss

```
@@ -0,0 +1,16 @@
+.ocs-shortcut-table {
+  & td {
```

Isn't this the same as...?

```suggestion
  td {
```

The BEM thing would be to give the element a name like `ocs-shortcut-table__key`

---

Habit of mine to always use `&` to be explicit.

I've updated the CSS now to use BEM.